app/src/main/java/com/lumiyaviewer/lumiya/voice/VivoxMessageQueue.kt

```
@@ -41,28 +40,28 @@ class VivoxMessageQueue {
                 vx_message_base_t vx_message_base_t2
                 if ((vx_message_base_t2 = VxClientProxy.get_next_message_no_wait()) != null) {
-                    Debug.Printf("Voice: got message from Vivox", Any[0])
+                    Debug.Printf("Voice: got message from Vivox", Object[0])
                     return vx_message_base_t2
                 }
-                Debug.Printf("Voice: waiting for Vivox event", Any[0])
+                Debug.Printf("Voice: waiting for Vivox event", Object[0])
                 try {
                     this.messageLock.wait()
                 }
                 catch (Exception e) { // Decompiler artifact - empty catch block }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)